eng/Version.Details.xml

```diff
@@ -1,16 +1,16 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.NET.Sdk" Version="10.0.100-preview.6.25321.102">
+    <Dependency Name="Microsoft.NET.Sdk" Version="10.0.100-preview.6.25326.107">
```

Comment:
👍 This looks like the same version we tested here:
* https://github.com/dotnet/android/pull/10246